script/gen_requirements_all.py

```diff
@@ -63,6 +63,7 @@
     'influxdb',
     'libpurecoollink',
     'libsoundtouch',
+    'locationsharinglib',
```

Comment:
There are no tests for this platform, right?

Reply:
Can't figure out how to pass the Tarvis tests :(